src/App.test.tsx

```diff
@@ -1,8 +1,8 @@
-import React from "react";
```

Comment:
Ouch, looks like I didnt add a prettier config file. could you pls merge it in and reformat the files so we can limit the changes that get merged?